### PR TITLE
Add dark mode vim-plug logo and use correct logo version in README

### PR DESCRIPTION
**Pull request by rnwst**
The vim-plug logo in the README wasn't visible in dark mode due to black lettering on a dark background. This PR fixes this by adding an color-inverted version of the logo, and using the right version in the README depending on the result of the `prefers-color-scheme` CSS media query.

### CODE REVIEW

**Review by junegunn — CHANGES_REQUESTED** [review]
Thanks!

Let's not rename plug.png to keep the image URL accessible.

File: README.md [hunk 1]
```diff
@@ -16,7 +16,14 @@
...
 
-<img src="https://raw.githubusercontent.com/junegunn/vim-plug/master/plug.png" height="75" alt="vim-plug">[![build](https://img.shields.io/github/actions/workflow/status/junegunn/vim-plug/test.yml?branch=master)](https://github.com/junegunn/vim-plug/actions/workflows/test.yml?query=branch%3Amaster)
+<picture>
+  <source media="(prefers-color-scheme: dark)" srcset="./plug-dark.png">
+  <img src="./plug.png" height="75" alt="vim-plug">
+</picture>
+<a href="https://github.com/junegunn/vim-plug/actions/workflows/test.yml?query=branch%3Amaster">
+  <img src="https://img.shields.io/github/actions/workflow/status/junegunn/vim-plug/test.yml?branch=master">
+</a>
+
```
Comment (junegunn) [inline README.md:26]:
```suggestion
```

Comment (junegunn) [reply README.md]: Hmm, it's still there.

Comment (junegunn) [reply README.md]: The triple sign indicates that the line above is an h1 header. If we just remove it, we'll lose the underline below the image.

image


So I replaced it with an actual h1 tag.

File: README.md [hunk 2]
```diff
@@ -14,10 +14,17 @@
 <br>
 <hr>
 </div>
-<br>
-
-<img src="https://raw.githubusercontent.com/junegunn/vim-plug/master/plug.png" height="75" alt="vim-plug">[![build](https://img.shields.io/github/actions/workflow/status/junegunn/vim-plug/test.yml?branch=master)](https://github.com/junegunn/vim-plug/actions/workflows/test.yml?query=branch%3Amaster)
-===
+<be>
```
Comment (junegunn) [inline README.md:17]:
```suggestion
<br>
```

File: README.md (hunk 1 above)
Comment (rnwst) [reply README.md]: Understood, sorry about that.

**Comment by rnwst** [comment]
Ready to be merged?

**Comment by junegunn** [comment]
Merged, thanks!

**Pull request merged**